tests/integration/tables/file.cpp

```diff
@@ -207,19 +213,31 @@ TEST_F(FileTests, test_sanity) {
     ASSERT_EQ(row.at("directory"), directory.string());
     ASSERT_EQ(row.at("filename"), test_file_name);
 
-    if (isPlatform(PlatformType::TYPE_WINDOWS)) {
-      auto link_path = boost::filesystem::path(expected_path);
-
-      if (row.at("path").rfind(".lnk") != std::string::npos) {
-        EXPECT_EQ(row.at("shortcut_target_path"),
-                  link_path.replace_extension(".lnk").string());
-        EXPECT_EQ(row.at("shortcut_target_type"), "File");
-        EXPECT_EQ(row.at("shortcut_target_location"), test_file_name);
-        EXPECT_EQ(row.at("shortcut_target_start_in"), directory.string());
-        EXPECT_EQ(row.at("shortcut_target_run"), "Normal window");
-        EXPECT_EQ(row.at("shortcut_target_comment"), "Test shortcut");
-      }
+#ifdef WIN32
+    {
+      // Check for corresponding shortcut (.lnk) files
+      auto link_index = getRowIndexForFileName(data, test_file_name + ".lnk");
+      ASSERT_TRUE(link_index.has_value());
+      const auto& row = data.at(link_index.value());
+
+      auto short_path = directory.string();
```

Comment:
Here we need to first go from UTF8 -> UTF16 with `stringToWString`.
But we also need the name of the file here; they can be shortened too.